docs/source/releases/latest/numpy-standards.yaml

```diff
@@ -0,0 +1,13 @@
+documentation:
+- description: |
+    Throughout GeoIPS, we have inconsistent coding standards.
+    The numpy docstring format is an external standard we already use,
+    and so we want that to be explicit in our documentation.
```

Comment:
https://github.com/NRLMMD-GEOIPS/geoips/blob/main/docs%2Fsource%2Fdevguide%2Fdocumentation_strategy.rst#L23

This was fairly explicit in our documentation strategy - are we just consolidating this in the new documentation format?

Reply:
This just provides some examples for what these docstrings should look like since the numpy docs have a lot more cases than we normally use. This is just for guidance.